README.md

```diff
@@ -19,36 +20,53 @@
 
 See also (related specs):
 
-*
+* https://w3c-ccg.github.io/vdl-vocab
+* https://github.com/w3c-ccg/vdl-vocab
 
 ## Install
 
-Requires Node.js 12+
+Requires Node.js 18+
 
 To install via NPM:
 
 ```
-npm install context-template
+npm install @digitalbazaar/vdl-context
 ```
 
 ## Usage
 
 ```js
-import exampleCtx from 'context-template';
-// or
-const exampleCtx = require('context-template');
-const {contexts, constants, appContextMap} = exampleCtx;
-
-exampleCtx.CONTEXT_URL
-// 'https://w3id.org/example/v1'
+// ESM
+import {
+  CONTEXT,
+  CONTEXT_URL,
+  constants,
+  context
+} from '@digitalbazaar/vdl-context';
+
+// CommonJS
+const ctx = require('@digitalbazaar/vdl-context');
+const {
+  CONTEXT,
+  CONTEXT_URL,
+  constants,
+  context
+} = ctx;
+
+// Default context URL
+CONTEXT_URL
+// -> 'https://w3id.org/vdl/v1'
+
+// Default context data
+CONTEXT
+// -> full context object
 
 // Codec term map value for CBOR-LD
-exampleCtx.constants.CBORLD_CODEC_VALUE
-// 0x0..
+constants.CBORLD_VALUE
+// -> 0x20
```

Comment:
This is not arbitrary.  Show as zero or register a new one and fix constants file.  See https://github.com/digitalbazaar/vdl-context/issues/6.